_opensearch/segment-replication/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Segment Replication Feature
+nav_order: 122
+parent: Opensearch
+has_children: true
+redirect_from:
+  - /opensearch/segment-replication/
+---
+
+## Segment Replication
+
+Opensearch 2.3 contains an experimental version of segment replication.
+
+With segment replication, segment files are copied across shards instead of indexing documents on each shard copy. This improves indexing throughput and lowers resource utilization at the expense of increased network utilization.
+
+As an experimental feature, segment replication will be behind a feature flag and must be enabled on **each node** of a cluster in addition to passing a new setting during index creation.
+{: .note }
+
+### Potential use cases
+
+1. Users who have very high writes load, but do not have high search requirements and are ok with longer refresh times.
+1. Users with very high loads who want to add new nodes, as you do not need to index all nodes when adding a new node to the cluster.
+1. This is the first step in a series of features to decouple reads and writes so resulting in a lower cost of compute.
```

Comment:
Should this one not be in the list? It is a general statement that applies to both. I would make it a bullet list because there will be only 2 items, and I would add the last item as a statement below. Also, did you mean "...reads and writes to lower compute cost"? There is a rogue "so" in the last sentence.

Reply:
I did yeah, recovered an old notes segment